lightwood/data/statistical_analysis.py

```diff
@@ -94,8 +116,10 @@ def statistical_analysis(data: pd.DataFrame,
         elif dtypes[col] in (dtype.integer, dtype.float, dtype.array):
             histograms[col] = get_numeric_histogram(filter_nan(df[col]), dtypes[col])
             buckets[col] = histograms[col]['x']
+        elif dtypes[col] in (dtype.date, dtype.datetime):
+            histograms[col] = get_datetime_histogram(filter_nan(df[col]), dtypes[col])
         else:
-            histograms[col] = {'x': [], 'y': []}
+            histograms[col] = {'x': ['Unknown'], 'y': [1]}
```

Comment:
Do not arbitrarily set 'y' to 1. At least make it None.

Additionally may make sense to set 'x' to None also.